src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    padding: 23px 0;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $main-text-color;
+    font-size: 12px;
+    line-height: 1.2;
+    font-weight: 500;
+
+    &:hover {
+      color: $link-color;
+    }
+
+    &:active {
+      color: $link-color;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
```

Comment:
It is better set this style for all links - to only for `is-active`, as it may lead to potential errors